src/make_particle.c

```diff
@@ -1810,6 +1814,42 @@ void InitShape(void)
 			Nmat_need=2;
 			break;
 		}
+	 case SH_SUPERELLIPSOID: {
+		 double aspectY, aspectZ;
+
+		 // Read and test parameters
+		 aspectY=sh_pars[0];
+		 TestPositive(aspectY,"aspect ratio y/x");
+		 aspectZ=sh_pars[1];
+		 TestPositive(aspectZ,"aspect ratio z/x");
+		 se_e=sh_pars[2];
+		 TestRangeII(se_e,"superellipsoid exponent e",0,10);
```

Comment:
It seems that there are no reasons to disallow e and n larger than 10. They do have problem with discretization of the scatterer (may result in zero dipoles), but that is natural. We only need to ensure that calculation of shape parameters work fine.